test/test_cli.py

```diff
@@ -13,7 +13,36 @@
 WHOAMI_RESPONSE = {
     "account": "myaccount"
 }
+VAR_HELP = '''Usage:
+     conjur [global options] variable <subcommand> [options] <VARIABLE_ID> <VALUE>
 
+positional arguments:
```

Comment:
same applies here as written [above](https://github.com/cyberark/conjur-api-python3/pull/118/files#r549070849). We already know we want to change the titles to `Commands` instead of `positional arguments` so let's refrain from hardcoding and just use assertRegex or assertIn instead of assertEqual